src/lib/__tests__/Fullscreen-test.js

```diff
@@ -77,17 +82,6 @@ describe('lib/Fullscreen', () => {
 
             expect(fullscreen.emit).to.have.been.calledWith('exit');
         });
-
-        it('should be called only once when the fullscreenchange event is emitted', () => {
```

Comment:
@DanDeMicco I deleted this test because 1. it stopped passing for me 😢and I couldn't get it working again but also 2. it seems like it's more testing the DOM than our code? Thoughts?

Reply:
nvm got it working again!